vimeo-networking/src/main/java/com/vimeo/networking/model/search/SearchFacetCollection.java

```diff
@@ -35,27 +35,62 @@
  * <p/>
  * Created by zetterstromk on 6/27/16.
  */
+@SuppressWarnings("unused")
 public class SearchFacetCollection implements Serializable {
 
     private static final long serialVersionUID = 3340976215489066653L;
 
     @Nullable
     @GsonAdapterKey("type")
     public SearchFacet mTypeFacet;
```

Comment:
Unfortunately no, Stag needs them to be public. There is hope to make Stag work with package private members in the future, but for now, they must be public.